docs/home/contribute/style-guide.md

```diff
@@ -144,14 +142,43 @@ The output is similar to this:
     NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
     nginx    1/1       Running   0          13s    10.200.0.4   worker0
 
+### Versioning Kubernetes examples
+
+Example that include version specific information should be matched with accompanying text and the relevant Kubernetes version identified.
+This can be done by verifying the relevant page has the Jekyll include statement:
+
+    `{% include task-tutorial-prereqs.md %}`
+
+And the front-matter YAML of the document includes the key `min-kubernetes-server-version` set to the relevant version.
+
+If the example YAML is in a standalone file, please review the markdown that includes it as a reference and verify that file has the appropriate version information.
```

Comment:
Our style guide says not to use "please".

Reply:
my mom will be hurt by this one ;-)